recipes/latexdiff/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "latexdiff" %}
+{% set version = "1.3.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ftilmann/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 3f066741414d502a2cc85005fc7f2f2fb310a627290e5b4f0b7eb8bdb3829a4f
+
+build:
+  number: 0
+  noarch: generic
+  script:
+    - cp {{ name }} $PREFIX/bin/
+
+requirements:
+  build:
+    - make
+  host:
+    - perl
+  run:
+    - perl
+    - perl-algorithm-diff
+
+test:
+  commands:
+    - test -f $PREFIX/bin/{{ name }}  # commands like --version have exit-code 1
```

It fails because it (correctly) writes on stderr, this should work:
```suggestion
    - latexdiff --version 2>&1 | grep 1.3.1.1
```